src/main/java/gregtech/common/terminal/app/console/ConsoleApp.java

```diff
@@ -27,7 +28,11 @@ private IGregTechTileEntity getMTE() {
 
     @Override
     public AbstractApplication initApp() {
-        if (getMTE() == null) { // 333 232
+        IGregTechTileEntity MTEResult = getMTE(); // Moved to make single call
```

Comment:
you have to use [lowerCamelCase](https://github.com/GregTechCEu/GregTech/blob/master/CODESTYLE.md) to name variables

Reply:
Felt more in line with the MTE in the method call, but fair enough. Forgot that bit lol.